src/test_cli/test_cli_parsing.py

```diff
@@ -0,0 +1,92 @@
+from collections import namedtuple
+from click.testing import CliRunner
+import pytest
+import desbordante
+
+from src.cli import desbordante_cli, ALGOS
+
+# There is a problem with get_opts() method for this algos
+UNFIXED_ALGOS = ['tane', 'pfdtane', 'cords', 'spider', 'faida', 'fd_first', 'split','naive_gfd_verifier',
+                 'gfd_verifier', 'egfd_verifier', 'apriori', 'naive_pfd_verifier']
+
+DATA_OPTS = ['table', 'tables', 'difference_table']
+SKIPPED_OPTS = ['tables_list', 'tables_directory']
+
+OptionInfo = namedtuple('OptionInfo', ['str', 'value'])
+
+OPTION_VALUES = {
+    (int,): OptionInfo('4', 4),
+    (float,): OptionInfo('0.1', 0.1),
+    (bool,): OptionInfo('True', True)
+}
+
+UNCOMMON_OPTIONS = {
```

Comment:
UNCOMMON sounds a bit obscure
```suggestion
ALGORITHM_SPECIFIC_OPTS = {
```